data_structures/binary_tree/wavelet_tree.py

```diff
@@ -0,0 +1,226 @@
+from typing import List, Optional
```

Comment:
```suggestion
from typing import Optional
```
In Python 3.9 and later, we can use `list` instead of `List`.

Reply:
Done, thanks